testing/util/BUILD.bazel

```diff
@@ -15,7 +15,6 @@ go_library(
         "deposits.go",
         "helpers.go",
         "merge.go",
-        "premine-state.go",
```

Comment:
I moved this for the same reason as `genesis.go` - the `testonly` bazel constraint prevented its use from non-test code.